style.css

```diff
@@ -0,0 +1,537 @@
+  font-size: 3rem;
+  color: #ec5242;
+  font-weight: 700;
+}
+
+.body-disc2 {
+  margin-top: 2rem;
+  font-size: 1.5rem;
+  line-height: 2.5rem;
+  border: 1px solid #fff;
+  padding: 1.5rem;
+  background: rgb(243, 241, 241);
+}
+
+.heading2 {
+  font-size: 2.7rem;
+  padding-top: 2rem;
+  color: #ec5242;
+}
+
+.home-footer {
+  font-size: 2rem;
+  padding: 1rem 1rem 8rem 1rem;
+}
+
+.main-program {
+  background: #272a31;
+  font-family: 'Lato', serif;
+  padding-bottom: 2rem;
+}
+
+.program-heading {
+  color: #fff;
+  padding: 2rem 0 1rem 0;
+  font-size: 1.8rem;
+  text-align: center;
+}
+
+.program-under {
+  height: 1px;
+  margin: 1rem 46%;
+  background-color: #ec5242;
+}
+
+.method-list {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding: 1rem;
+}
+
+.method-container {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding: 0.5rem;
+}
+
+.method-item {
+  display: flex;
+  gap: 2rem;
+  justify-content: center;
+  align-items: center;
+  padding: 2rem;
+  background: #4a4e52;
+}
+
+.method-item i {
+  color: #fff;
+  font-size: 3rem;
+}
+
+.method-item-title {
+  color: #ec5242;
+  font-size: 1.9rem;
+}
+
+.method-item-disc {
+  color: #fff;
+  font-size: 1.3rem;
+}
+
+.program-btn-container {
+  display: flex;
+  justify-content: center;
+}
+
+.program-btn {
+  background: #ec5242;
+  color: #fff;
+  font-size: 1.7rem;
+  padding: 1rem 3rem;
+  border: none;
+}
+
+.program-disc {
+  display: none;
+}
+
+.teachers {
+  background: #fff;
+  font-family: 'Lato', serif;
+}
+
+.teacher-heading {
+  font-size: 2rem;
+  text-align: center;
+  padding: 2rem 2rem 0 2rem;
+}
+
+.teacher-under {
+  height: 1px;
+  margin: 1rem 46%;
+  background-color: #ec5242;
+}
+
+.teachers-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.teacher-body {
+  display: flex;
+  padding: 2rem;
+  gap: 1rem;
+  font-family: 'Lato', serif;
+}
+
+.teacher-img-container {
+  width: 50vw;
+}
+
+.teacher-img {
+  width: 80%;
+}
+
+.teacher-disc-body {
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  padding: 1rem 0;
+  flex: 1;
+}
+
+.teacher-title {
+  font-size: 2rem;
+}
+
+.teacher-disc {
+  font-size: 1.2rem;
+  color: #ec5242;
+}
+
+.teacher-disc2 {
+  font-size: 1.2rem;
+}
+
+.footer {
+  display: none;
+}
+
+.copyright {
+  display: none;
+}
+
+@media only screen and (min-width: 768px) {
+  html {
+    font-size: 80%;
+  }
+
+  .nav {
+    display: none;
+  }
+
+  .header {
+    display: flex;
+    font-family: 'Lato', serif;
+    justify-content: flex-end;
+    align-items: center;
+    padding: 0 9rem 0 0;
+    gap: 2rem;
+    background: #3e3e3e;
+  }
+
+  .header i {
+    font-size: 1.5rem;
+    color: #fff;
+    padding: 0.5rem 0;
+  }
+
+  .header a {
+    text-decoration: none;
+    font-size: 1.5rem;
+    color: #fff;
+  }
+
+  .dnav {
+    display: flex;
+    background: #fff;
+    justify-content: center;
+    align-items: center;
+    gap: 150px;
+    padding: 0 198px 0 209px;
+  }
```

Comment:
- [ ] Good job on implementing the mobile-first approach. However, there is a little issue with your header. When the screen width gets smaller, it doesn't fit the full view width and thus causes horizontal scrolling on the page.
![image](https://user-images.githubusercontent.com/91486791/235133741-fa008101-844d-4fc2-9ab6-a6d555280c34.png)